src/test/scala/pick16SCandidates.scala

```diff
@@ -106,13 +106,21 @@ case object pick16SCandidates extends FilterData(
       if (commonID != fasta.getV(header).id)
         sys.error(s"ID [${commonID}] is not found in the FASTA. Check RNACentral filtering.")
 
+      /* This predicate determines whether the *sequence* value is OK, and will be kept. */
+      val sequenceIsOK: String => Boolean =
+        seq: String =>
+          ( seq.length >= minimum16SLength )              &&
+          ( (seq.count(_ == 'N') / seq.length) <= 0.01 )  &&
+          ( !(seq containsSlice "NNNNNNNN") )
```

Comment:
I think that this is a case when omitting braces doesn't improve readability and makes code error-prone. Also, why `val`? This forces you to write the arg type twice? I will commit my suggestion here